### PR TITLE
Support show alluxio setup time

### PR DESCRIPTION
**Pull request by iluoeli**
### Ⅰ. Describe what this PR does
After the setup of Alluxio runtime, show its setup time in the status of runtime

### Ⅱ. Does this pull request fix one issue?

fixes issue #595 

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: api/v1alpha1/status.go [hunk 1]
```diff
@@ -104,6 +104,9 @@ type RuntimeStatus struct {
 	// +optional
 	FuseNumberAvailable int32 `json:"fuseNumberAvailable,omitempty"`
 
+	// The setup time of runtime in seconds
+	SetupPeriodSeconds *int64 `json:"setupPeriodSeconds,omitempty"`
```
Comment (cheyang) [inline api/v1alpha1/status.go:108]: I think it's better to provide readable duration. You can learn from `duration` time of K8s implementation.

**Review by cheyang — CHANGES_REQUESTED** [review]

File: pkg/ddc/alluxio/status.go [hunk 2]
```diff
@@ -130,6 +130,12 @@ func (e *AlluxioEngine) CheckAndUpdateRuntimeStatus() (ready bool, err error) {
 			ready = true
 		}
 
+		// Update the setup time of Alluxio runtime
+		if ready && runtimeToUpdate.Status.SetupPeriodSeconds == nil {
```
Comment (cheyang) [inline pkg/ddc/alluxio/status.go:134]: @yangyuliufeng I remember we have the similar function. Could you help @iluoeli refactor this implementation?

**Review by cheyang — CHANGES_REQUESTED** [review]

File: api/v1alpha1/status.go (hunk 1 above)
Comment (iluoeli) [reply api/v1alpha1/status.go]: done

**Review by yangyuliufeng — APPROVED** [review]

File: pkg/ddc/alluxio/status.go (hunk 2 above)
Comment (yangyuliufeng) [reply pkg/ddc/alluxio/status.go]: > @yangyuliufeng I remember we have the similar function. Could you help @iluoeli refactor this implementation?

OK

**Comment by iluoeli** [comment]
@yangyuliufeng  will create a new pr, so this pr is closed.

**Pull request closed without merging**